components/Tweet.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <v-container>
+    <v-row no-wrap no-gutters>
+      <v-col>
+        <v-row>
+          <v-col class="d-flex justify-center align-center">
+            <v-card-title justify-center>
+              <v-avatar color="cyan lighten-2 avatar-border" size="100">
+                <v-img
+                  class="elevation-6"
+                  :alt="tweet.name"
+                  :src="tweet.profile_image_url_original"
+                ></v-img>
+              </v-avatar>
+            </v-card-title>
+          </v-col>
+        </v-row>
+        <v-row>
+          <v-col class="d-flex justify-center">
+            <v-card-text class="text-body-1 tweet-content" height="400px">
+              {{ tweet.text }}
+            </v-card-text>
+          </v-col>
+        </v-row>
+        <v-row>
+          <v-col class="d-flex justify-center align-center">
```

Comment:
design nit: in the future, instead of using css classes that are 1 line css rules, it would be preferrable to remove the obfuscation and just use the actually css rule. 

for some context, @renderf0x, this will make it easier if we shift towards a new design system as we will probably use more sass variables which we cannot take advantage of using this current approach. 

for example:
instead of `d-flex`, just use `display:flex`.

Reply:
I can change this, but note that so long a we're using Vuetify, this is a standard pattern, and one that is official guidance for things like [controlling breakpoints](https://vuetifyjs.com/ja/styles/display/) (admittedly not what we're doing here).

Given our usage of Vuetify components I think we'll need to do a major rewrite if we were ever going to move away, as even the tag class `<v-col>` here is specific to that framework.

Having said that, I can change it if you prefer.

Reply:
I would recommend changing it. See above comment.

Reply:
sure.

Reply:
As a note here, I'm moving `d-flex` to a specific class to apply `display: flex;` to these, but the other classes (`justify-center align-center`) actually need to stay, despite being conveniences, and here's why - 

In their 'infinite' wisdom, the implementers of Vuetify have chosen to sprinkle `!important` directives throughout their classes on columns and the like, and even adding something like `justify-content: center` directly to the element doesn't take effect unless using the `!important` alongside it. These convenience classes add that, and I'd rather use that in this fashion than have us start actually using `!important` in our codebase.

I think it's really dumb that it was used at all in Vuetify; seems very unprofessional tbh.

Reply:
Is there any way around the `!important`s? Can we narrow the scope of our css better for it to apply correctly? 
If not, that's fair for now.  (I personally would still prefer seeing the actual css rule with `!important` over realizing css is also coming from classes, but I'll leave this one to you)

Reply:
Unfortunately, no. I added it directly to the element and it still got clobbered. Pretty dumb of them to use these but here we are =(.